app/assets/javascripts/stradivari/helpers.js

```diff
@@ -40,6 +40,27 @@ window._TABLE_ = (function() {
         concat(params)
       );
 
+    },
+
+    // example
+    // in: http://mysite.com?stradi_tabs[]=bar&stradi_tabs[]=baz&foo=bar
+    // out: {stradi_tabs: ['bar', 'baz'], foo: "bar"}
+    parseURLParameters: function(uri) {
+      var re = /([^&=]+)=?([^&]*)/g;
+      var decodeRE = /\+/g;  // Regex for replacing addition symbol with a space
+      var decode = function (str) {return decodeURIComponent( str.replace(decodeRE, " ") );};
+      var query = uri.split('?')[1]
```

Comment:
What about using a 3rd party js library to deal with URIs? like http://medialize.github.io/URI.js/